weblab_cg/data/tests/weblab_model.pyx

```diff
@@ -163,6 +163,7 @@ cdef class TestModel(CvodeSolver):
         # Mapping from qualified names to state indices; only for states that
         # have a variable name.
         self.state_var_map = {}
+        self.state_var_map['membrane_voltage'] = 0
```

Comment:
You could just initialize it in 1 go:
`self.state_var_map = {'membrane_voltage': 0}

Reply:
That's generated code :-D

There can be zero or multiple state vars in that map, and trying to match a template :-)